ROADMAP.md

```diff
@@ -1,177 +1,94 @@
-# Kubeflow Pipelines Roadmap
+### KFP ROADMAP
 
-## Kubeflow Pipelines 2022 Roadmap
+The purpose of this KFP Roadmap document is to outline the planned themes and items for future development and improvements of Kubeflow Pipelines (KFP). It categorizes these efforts into several key areas: User Experience, User Documentation, Core Platform Stability, Extensibility, Integrations, and Quality of Life, QE, Performance & Scalability, Security & Compliance, and Community & Governance. The document also provides information on how to contribute to KFP.
 
-### KFP v2
+## New Features & Enhancements
 
-Design: [bit.ly/kfp-v2](https://bit.ly/kfp-v2)
+Q4 2025
```

Comment:
Could we be a bit more vague and have the sections be 2025 and 2026 rather than specific quarters? We can also include specific goals/themes of what we're trying to achieve rather than a checklist of features since to be honest, we don't know what we'll be delivering a year from now.